src/Compiler/Microsoft.CodeAnalysis.Razor/src/TagHelperTypes.cs

```diff
@@ -11,7 +11,7 @@ internal static class TagHelperTypes
 
     public const string IComponent = "Microsoft.AspNetCore.Components.IComponent";
 
-    public const string IDictionary = "System.Collections.Generic.IDictionary`2";
+    public const string IDictionary = "System.Collections.Generic.IDictionary<TKey, TValue>";
```

Comment:
Is there a reason this name isn't in ComponentApi.cs like the other type names?

Reply:
`TagHelperTypes` is the more general set of 'well known' things. `ComponentApi` just contains component specific stuff which is a sub-set of tag helpers